packages/roosterjs-editor-plugins/lib/plugins/Paste/Paste.ts

```diff
@@ -74,8 +77,10 @@ export default class Paste implements EditorPlugin {
             const { htmlAttributes, fragment, sanitizingOption, clipboardData } = event;
             const trustedHTMLHandler = this.editor.getTrustedHTMLHandler();
             let wacListElements: Node[];
-
-            if (isWordDocument(htmlAttributes)) {
+            const table = getWordOnlineTable(fragment);
+            if (table) {
+                wordOnlineTableConverter(this.editor, table);
```

Comment:
When paste from Word Online, can it fall into line 98 or any other existing branch?

If so, this should be moved into that branch, otherwise it will cause that branch not executed.